tests/__serialize_005.phpt

```diff
@@ -5,8 +5,11 @@ __serialize() mechanism (005): parent::__unserialize() is safe
 --FILE--
 <?php
 
+// NOTE: PHP 8.1.0 changed the way property iteration is done to optimize memory usage(affects var_dump output order but not the correctness of igbinary).
```

Comment:
@derrabus can this be related to https://github.com/symfony/symfony/issues/41592 ?

Reply:
Yes!